src/types/value.rs

```diff
@@ -26,6 +27,13 @@ pub struct Value {
     pub value_type: ValueType,
 }
 
+impl Into<Value> for String {
```

Comment:
It's a good idea, that could simplify the work flow.
Also I'm going to create a separate issue to have `Into<Value>` implementation bunch of other types like integers, vectors etc.